docs/abtest.rst

```diff
@@ -8,6 +8,39 @@
 A/B Testing
 ===========
 
+Nimbus experiments
+------------------
+
+Nimbus (Experimenter), is Mozilla's 1st-party experimentation platform.
+Experiments from this platform will split traffic before users arrive on
+the page (meaning no client-side JS delay).
+
+.. Note::
+
+    Bedrock code will still determine what templates to show based on locales
+    and the url parameters added by Nimbus.
+
+There are generally 3 user groups to consider:
+
+#. In experiment - control branch
+#. In experiment - treatment branch
+#. Not in experiment
+
+The Firefox team will set up the experiment and provide the URL params:
+
+- **branch** (required): identifies an experiment. When there is a single experiment, the branch value can be used to distinguish control and treatment (i.e. ``experiment-wnp-136-control`` and ``experiment-wnp-136-vpn``)
+
+- **variant** (optional): when multiple experiments run at the same time, this secondary param distinguishes control and treatment variants within each experiment (i.e. ``experiment-wnp-136-vpn`` has 2 variants and ``experiment-wnp-136-tabs`` has 4 variants). Nimbus variant must match pre-defined value (i.e. ``v1``, ``v2``, ``v3``, ``v4``).
```

Comment:
in Nimbus, this is covered in branches, but the branch URLs include a variant param that Bedrock uses

in Bedrock, this is where we identify the experiment branches as v1 (control), v2 (treatment-a), etc.